engine/class_modules/warlock/sc_warlock.cpp

```diff
@@ -761,7 +761,7 @@ void warlock_t::init_procs()
   procs.demonic_calling = get_proc( "demonic_calling" );
   procs.soul_conduit    = get_proc( "soul_conduit" );
   procs.corrupting_leer = get_proc( "corrupting_leer" );
-
+  procs.prolonged_decimation = get_proc( "prolonged_decimation" );
```

Comment:
this is a small thing, but clang-format here would want to line all these `procs` lines up by the `=`

Reply:
Thanks for pointing it out, I thought it would apply automatically after setting CLANG up. Now just used it for the sections with my changes manually.

Reply:
I am not sure if the warlock module uses clang formatting. I think the priest module is one of the few which nowadays tries to use it somewhat consistently and if possible all the time. Which I think is great since I love reading formatted code but hate writing it :-)

But yeah for warlock just formatting the lines you worked on is probably the best, and it might need discussion with @Azevara if you guys want to do more formatting of existing code. It should in any case be in a separate commit and PR so as not to hide functional code changes.

Reply:
Clang formatting is inconsistent in the module and any attempt to apply it right now has a high chance of creating a messy commit, so just something to be aware of. We'll need to revisit that issue once all the Shadowlands code is in.